client/webserver/webserver.go

```diff
@@ -625,6 +627,37 @@ func New(cfg *Config) (*WebServer, error) {
 	return s, nil
 }
 
+// fetchLatestVersion is a helper function to retrieve the latest version of the app
+// from github.
+func (w *WebServer) fetchLatestVersion(ctx context.Context) {
+	if w.latestVersion != "" {
+		return // already set
+	}
```

Comment:
Should continue checking anyway?

Reply:
Releases don’t happen very often.
User would usually restart their client every now and then.

But, I could add a paycheck timestamp and recheck once it exceeds 24hrs?(if the server stays up for that long)